.travis.yml

```diff
@@ -62,6 +62,7 @@ script:
           coverage run --branch --source tlsfuzzer -m unittest discover;
       fi
   - coverage report -m
+  - python verify-scripts-json.py
```

Comment:
Another improvement could be to simplify the script so that it is run twice, as:
```
python verify-scripts-json.py tests/tlslite-ng.json
python verify-scripts-json.py tests/tlslite-ng-random-subset.json
```
Although that seems like a waste it has the advantage that I, for example, can use it to test using another file in my home directory. That's actually a common case as it is expected for one to write such a json file for other implementations than tlslite.

Reply:
Yes it makes sense, fixed that